mappings/net/minecraft/client/texture/Sprite.mapping

```diff
@@ -60,3 +68,17 @@ CLASS net/minecraft/class_1058 net/minecraft/client/texture/Sprite
 		FIELD field_21758 images [Lnet/minecraft/class_1011;
 		METHOD <init> (Lnet/minecraft/class_1058;Lnet/minecraft/class_1058$class_4727;I)V
 			ARG 3 mipmap
+		METHOD method_24128 apply ()V
+			COMMENT Linearly interpolate between the current and next frame on all miplevels
+			COMMENT based on the tick position within the current frame,
+			COMMENT and upload the results to the currently bound texture to the frame slot at position (0,0).
+		METHOD method_24129 lerp (DII)I
```

Comment:
So this is lerping by each r/g/b portion? maybe mention that to clarify from regular lerp, like add javadoc or call this `lerpColor`

Reply:
No, not at all, this is a purely value(int)-level lerp that doesn't know about colors or anything. The code that uses it though applies it to RGB components.

Reply:
Hmm, great. Think mathhelper only has int lerp. I saw that jd is for apply so i suggested "add javadoc" to this method, which now appears unnecessary.

Reply:
Oh sorry I misread your comment earlier. I will add javadoc to lift the confusion then.

Reply:
@liach if you could review whether the javadoc I added is adequate I'd be grateful =3